src/tests/suite/io_conf/daos_generate_io_conf.c

```diff
@@ -454,6 +454,15 @@ generate_io_conf_rec(int fd, struct current_status *status)
 	eph = status->cur_tx;
 	inject_fail_idx = rand() % epoch_times;
 	tgt = rand() % tgt_size;
+	if ( tgt_size != -1 ) {
```

Comment:
(style)  space prohibited after that open parenthesis '('
(style)  space prohibited before that close parenthesis ')'